packages/core/src/lib/connection_manager.ts

```diff
@@ -38,33 +38,9 @@ export class ConnectionManager
 
   private currentActiveParallelDialCount = 0;
   private pendingPeerDialQueue: Array<PeerId> = [];
-  private online: boolean = false;
 
-  public isConnected(): boolean {
-    return this.online;
-  }
-
-  private toggleOnline(): void {
-    if (!this.online) {
-      this.online = true;
-      this.dispatchEvent(
-        new CustomEvent<boolean>(EConnectionStateEvents.CONNECTION_STATUS, {
-          detail: this.online
-        })
-      );
-    }
-  }
-
-  private toggleOffline(): void {
-    if (this.online && this.libp2p.getConnections().length == 0) {
-      this.online = false;
-      this.dispatchEvent(
-        new CustomEvent<boolean>(EConnectionStateEvents.CONNECTION_STATUS, {
-          detail: this.online
-        })
-      );
-    }
-  }
+  private isConnectedToNetwork: boolean = false;
+  private isConnectedToWakuNetwork: boolean = false;
```

Comment:
imo it's not immediately clear what being connected to network means vs being connected to waku network. might be worth adding a comment

Reply:
renamed to `isConnectedToInternet` and `isConnectedToP2PNetwork` 
or you have better naming in mind?

Reply:
even better, removed those properties